data/variants.txt

```diff
@@ -8,6 +8,102 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+
+ship "Behemoth" "Behemoth (Scavenged)"
```

Comment:
Would be better as `(unarmed)` as that is more descriptive of what the actually are.